code/modules/transport/pods/secondary_system.dm

```diff
@@ -585,6 +600,30 @@
 			src.ship.myhud.update_states()
 		boutput(usr, SPAN_NOTICE("Thrusters will now provide ship movement to the [src.turn_dir]."))
 
+/obj/item/shipcomponent/secondary_system/thrusters/afterburner
```

Comment:
It'd be really nice if any new pod components could come with a sprite, we have a few too many using the default cargo hold sprite right now and it's building up a lot of art-debt for the future. It doesn't have to be anything complex, just more than a featureless cylinder!